test/controllers/calls_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class CallsControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  def setup
+    @user = build :user
+    sign_in @user
+    @case = create :case
+    @call = build :call
+  end
+
+  describe 'create method' do
+    before do
+      @call = attributes_for :call
+    end
+
+    it 'should create and save a new call' do
+      assert_difference 'Case.find(@case).calls.count', 1 do
+        post :create, status: @call.status, id: @case
+      end
+      assert_response :redirect
+    end
+
+    # If we have to look up the call through a case this test seems redundant
+    # I.e. I think the previous test also confirms the association
+    # it 'should associate the new call with a case' do
```

Comment:
I agree that this is a little redundant as written (it's much more relevant in postgres / relational models), but we do want to test that the controller is saving the object to the DB the way we think it is. Maybe we remix this to check equality of the status, and that the parent case ids are the same? (I think keep the `assert_includes` line as well, great call on that.)